src/man/kiwix-serve.1

```diff
@@ -1,102 +1,89 @@
-.TH KIWIX 1 "21 May 2012"
+
+.TH KIWIX 1 "10 July 2023"
 
 .SH NAME
 kiwix-serve \- Kiwix HTTP Server
 
 .SH SYNOPSIS
 .B kiwix-serve
-[\fB-h\fR] [\fB-a\fR \fBPID\fR] [\fB-d\fR] [\fB-i\fR \fBADDRESS\fR] [\fB-M\fR]
-[\fB-m\fR] [\fB-n\fR] [\fB-b\fR] [\fB-p\fR \fBPORT\fR] [\fB-r\fR \fBURL_ROOT_LOCATION\fR]
-[\fB-s\fR \fBSEARCH_LIMIT\fR] [\fB-t\fR \fBTHREADS\fR] [\fB-v\fR] [\fB-V\fR]
-[\fB-z\fR] [\fB-c\fR \fBCUSTOM_INDEX\fR] [\fB-L\fR \fBIP_CONNECTION_LIMIT\fR]
-[\fB--library\fR] \fBLIBRARY_PATH\fR
-.br
-.B kiwix-serve
-[\fB-h\fR] [\fB-a\fR \fBPID\fR] [\fB-d\fR] [\fB-i\fR \fBADDRESS\fR] [\fB-M\fR]
-[\fB-m\fR] [\fB-n\fR] [\fB-b\fR] [\fB-p\fR \fBPORT\fR] [\fB-r\fR \fBURL_ROOT_LOCATION\fR]
-[\fB-s\fR \fBSEARCH_LIMIT\fR] [\fB-t\fR \fBTHREADS\fR] [\fB-v\fR] [\fB-V\fR]
-[\fB-z\fR] [\fB-c\fR \fBCUSTOM_INDEX\fR] [\fB-L\fR \fBIP_CONNECTION_LIMIT\fR]
-[\fBZIM_PATH\fR]...
+[\fB--library\fR] [\fB-h\fR] [\fB-i ADDR\fR] [\fB-p PORT\fR] [\fB-r ROOT\fR] [\fB-d\fR] [\fB-a PID\fR] [\fB-M\fR]
+[\fB-m\fR] [\fB-n\fR] [\fB-b\fR] [\fB-t N\fR] [\fB-s N\fR] [\fB-z\fR] [\fB-c PATH\fR] [\fB-L N\fR] [\fB-v\fR] [\fB-V\fR]
 
 .SH DESCRIPTION
 The \fBkiwix-serve\fR command is used to run a stand-alone HTTP server for serving ZIM contents over the network.
 
 .SH OPTIONS
 .TP
-\fB-h\fR, \fB--help\fR
-Print a help message.
-
-.TP
-\fB-a\fR, \fB--attachToProcess\fR \fBPID\fR
-Exit if the specified process ID (\fBPID\fR) is not running anymore.
+\fB--library\fR
+By default, kiwix-serve expects a list of ZIM files as command line arguments. Providing the --library option tells kiwix-serve that the command line argument is rather a library XML file.
 
 .TP
-\fB-d\fR, \fB--daemon\fR
-Detach the HTTP server daemon from the main process.
+\fB-h, --help\fR
+Print a help message.
```

Comment:
This is strange position for the `--help` option.

Reply:
I didn't get you here, can you elaborate?

Reply:
In the current version of the PR the `--help` option comes second in the option list which doesn't fit any logical scheme (alphabetical, historical, importance, etc). Please move it to the end.

Reply:
done